ios/FluentUI/Label/Label.swift

```diff
@@ -133,7 +129,7 @@ open class Label: UILabel, TokenizedControlInternal {
             return
         }
 
-        let labelFont = tokenSet[.font].uiFont
+        let labelFont = labelFont ?? tokenSet[.font].uiFont // shouldScale: adjustsFontForContentSizeCategory
```

Comment:
Where did this comment come from?

Reply:
I think this comment used to live here at some point (possibly when migrating away from `FontInfo` wrappers or something), although I don't remember the exact details. But yeah, I don't think we need this.